python/agents/application/prompts.py

```
@@ -0,0 +1,245 @@
+from typing import List
+from datetime import datetime
+
+
+class Prompter:
+
+    def generate_simple_ai_trader(market_description: str, relevant_info: str) -> str:
```

Pylint (3.3.7)

[error] Method 'generate_simple_ai_trader' should have "self" as first argument

(E0213)